nuopc.runconfig

```diff
@@ -278,19 +278,19 @@ MED_attributes::
 ::
 
 CLOCK_attributes::
-     atm_cpl_dt = 3600
+     atm_cpl_dt = 99999     #{model}_cpl_dt is not used as it is overridden by the overall time step in nuopc.runseq
```

Comment:
Could we add to every line something like `not used in ACCESS-OM3`

Reply:
Maybe just say "#not used" and people can do a git blame for details ?

Reply:
For all the unused variables?

Reply:
I guess so (we are talking about 7 lines) ?

Reply:
Added comments for all cpl time steps in this commit ec0b83b